rero/rero-ils
# user interface: translations

* NEW Updates the translations.
* FIX Ignores the angular builds when extracting messages to translate.
* FIX Adds a favicon for apple touche devices.

Signed-off-by: Igor Milhit <igor.milhit@rero.ch>

-----

### How to test

- `pipenv run invenio collect` (for the apple favicon), but you'll need to build the angluar app, so `./scripts/bootstrap` is the way...
- check the user interface (public and professional) with different locales
